pandas/tests/test_strings.py

```diff
@@ -1984,13 +1984,14 @@ def test_split_to_multiindex_expand(self):
         tm.assert_index_equal(result, exp)
         self.assertEqual(result.nlevels, 3)
 
-        idx = Index(['some_unequal_splits', 'one_of_these_things_is_not'])
-        result = idx.str.split('_', expand=True)
-        exp = MultiIndex.from_tuples([('some', 'unequal', 'splits', NA, NA, NA
-                                       ), ('one', 'of', 'these', 'things',
-                                           'is', 'not')])
-        tm.assert_index_equal(result, exp)
-        self.assertEqual(result.nlevels, 6)
+        with self.assertRaises(ValueError):
```

Comment:
The test constructs a multi-index with unbalanced tuples which now throws a ValueError. The split strings have different lengths and because expand=True, a MultiIndex is constructed; see a few lines below.  Therefore, I modified the test check for a ValueError.